website/docs/docs/build/udfs.md

```diff
@@ -0,0 +1,208 @@
+---
+title: "User-defined functions"
+description: "Learn how to add user-defined functions (UDFs) to your dbt projects."
+id: "udfs"
+---
+
+# User-defined functions <Lifecycle status="beta" />
+
+:::info
+UDFs are not yet supported in the dbt Fusion engine.
+:::
+
+User-defined functions (UDFs) enable users to define and register custom functions in your warehouse. Like [macros](/docs/build/jinja-macros), UDFs promote code reuse. They run natively in the warehouse so you can reuse the same logic in tools outside dbt.
```

Comment:
eventually, we will also support UDAFs and UDTFs - not sure if we want to generally refer to these as "functions" or "UDFs"

Reply:
Maybe functions in this intro paragraph, but going forward we have two kinds of UDFs.